test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -51,12 +106,32 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=/csi/csi.sock"
+            - "--http-endpoint=:22013"
+            - "--leader-election"
+            - "--leader-election-namespace=$(PDCSI_NAMESPACE)"
             - "--handle-volume-inuse-error=false"
+          env:
+            - name: PDCSI_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          ports:
+            - containerPort: 22013
+              name: http-endpoint
+              protocol: TCP
+          livenessProbe:
+            failureThreshold: 1
+            httpGet:
+              path: /healthz/leader-election
+              port: http-endpoint
+            initialDelaySeconds: 10
+            timeoutSeconds: 10
+            periodSeconds: 20
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
         - name: gce-pd-driver
-          image: gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver:v1.0.1-gke.0
+          image: gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver:v1.2.1-gke.0
```

Comment:
We haven't fixed the k8s image building issue yet?

Reply:
Thanks, I just copied this from stable-master on the upstream driver. But that's out of date, we've moved to v1.2.2-gke.2 on GKE already.